paddle/fluid/operators/conv_mkldnn_op.cc

```diff
@@ -423,16 +427,20 @@ class ConvMKLDNNGradOpKernel : public paddle::framework::OpKernel<T> {
      * ('any') which lets a primitive (conv backward in this case) choose
      * the memory format preferred for best performance
      */
+    std::string data_format = ctx.Attr<std::string>("data_format");
+    auto chosen_memory_format =
+        platform::data_format_to_memory_format(data_format);
```

Comment:
These lines of code are the same, maybe this code can be moved to one function.

Reply:
This is just 2 lines of code excluding the required line-break. I think it would be too much.